Project.toml

```diff
@@ -6,6 +6,7 @@ version = "0.9.17"
 Adapt = "79e6a3ab-5dfb-504d-930d-738a2a938a0e"
 Atomix = "a9b6321e-bd34-4604-b9c9-b65b8de01458"
 ChainRulesCore = "d360d2e6-b24c-11e9-a2a3-2a2ae2dbcce4"
+FFTW = "7a1cc6ca-52ef-59f5-83cd-3a7055c09341"
```

Comment:
Can we make this a weak dependence and move any function relying on FFTW under an extension?

Reply:
Done